packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -55,7 +55,7 @@ interface StateProps {
   dh: DhType;
   isolatedLinkerPanelId?: string;
   links: Link[];
-  timeZone: string;
+  timeZone?: string;
```

Comment:
Shouldn't be optional
```suggestion
  timeZone: string;
```